src/bundle/Resources/public/scss/fieldType/edit/_ezimageasset.scss

```diff
@@ -0,0 +1,74 @@
+.ez-field-edit--ezimageasset {
+    .ez-field-edit-preview {
+        padding: 1.25rem 0 1.25rem 1.25rem;
+
+        &__media {
+            max-width: 100%;
+            width: auto;
+            height: auto;
+            display: block;
+        }
+
+        &__details {
+            color: $ez-color-base-light;
+            font-size: .75rem;
+            padding-top: .5rem;
+        }
+    }
+
+    &.is-invalid {
+        .ez-field-edit-preview {
+            position: relative;
+
+            &:before {
+                content: '!';
+                position: absolute;
+                top: 0;
+                left: -20px;
+                width: 20px;
+                background: red;
+                color: white;
+                text-align: center;
+                font-size: 1.5rem;
+                border-top-left-radius: 5px;
+                border-bottom-left-radius: 5px;
+            }
+        }
+    }
+}
+
+.ez-field-edit--ezimageasset {
+    &__preview-wrapper {
```

Comment:
I think this will be compiled to: `ez-field-edit--ezimageasset__preview-wrapper` and this is not correct usage of BEM